### ISSUE TITLE
FR: Close all Unnamed Tabs (Intruder/Reapter)

### ISSUE THREAD

**Issue by a-marionette** [mdsecresearch/BurpSuiteSharpener#48]
See title

**Comment by irsdl** [comment]
Yes this and also closing all the tabs that match a pattern is in my roadmap. The only issue is that default Burp closing options like `Re-open close tab` can become useless unless I reimplement it (I haven't found a solution that can do this without consuming too much memory but there must be a way). I normally open hundreds of tabs so I can sense when it goes really slow (for example when adding HackVertor when I have too many tabs)
